.github/workflows/main.yml

```
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Seems like the checks are no longer running on new pushes. Might be necessary to add this:

```suggestion
  pull_request:
    branches: [ main ]
    types: [ opened, synchronize ]
```

---

My bad, the merge conflict is what is preventing checks from being run.